00_core.ipynb

```diff
@@ -240,7 +256,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The default mapping from labels to markdown headings is:\n",
+    "Calling this method fetches relevant pull requests and issues from the GitHub API, and categorizes these entities according to a user-supplied mapping from labels to markdown headings.  The default mapping from labels to markdown headings is:\n",
```

Comment:
Most docs guidelines I've seen recommend removing phrases like "calling this method...". So you could start this sentence at "Fetches"